<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathWidget</class>
 <widget class="QWidget" name="PathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>37</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayoutMain">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QWidget" name="widgetPath" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonEdit">
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+L</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetPath</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
